sklearn/cross_validation.py

```diff
@@ -1029,29 +1032,34 @@ def cross_val_predict(estimator, X, y=None, cv=None, n_jobs=1,
     Returns
     -------
     preds : ndarray
-        This is the result of calling 'predict'
+        This is the result of calling 'decision_function'
```

Comment:
decision_function has a very specific meaning in scikit-learn and so I don't think we should use this term.